kurtosis_package/deployers/utils.star

```diff
@@ -32,11 +26,21 @@ def deploy_generic_contract(plan, context, deployment):
     root = "/app/" + contracts_path + "/"
 
     input_artifacts = generate_input_artifacts(plan, context, input, root)
-
     if is_remote_repo:
         deployer_img = gen_deployer_img(repo, deployment["ref"], contracts_path)
     else:
-        deployer_img = FOUNDRY_IMAGE
+        deployer_img = ImageBuildSpec(
+            image_name="Layr-Labs/foundry-local",
+            build_context_dir="../dockerfiles/",
+            build_file="contract_deployer.Dockerfile",
+            target_stage="foundry",
+            build_args={
+                "CONTRACTS_REPO": "",  # will skip git logic
+                "CONTRACTS_REF": "local",  # dummy value, required by Docker ARG
+                "CONTRACTS_PATH": contracts_path,
+            },
+        )
+
```

Comment:
Why not use `FOUNDRY_IMAGE`?

Reply:
If we use the foundry image, getting permission denied error for /app 
So giving permission in the docker file

Reply:
We can just leave the `FOUNDRY_IMAGE` constant in this file and `contract_deployer.Dockerfile` file as before then.

Reply:
https://github.com/Layr-Labs/avs-devnet/pull/221/commits/c57c4d4997dca350d3816973e392076e71213e5f
https://github.com/Layr-Labs/avs-devnet/pull/221/commits/17ce1997a383d3b54b357f1f4941b1ce8bf95fca
if you meant this

Reply:
I mean we can revert the changes on `kurtosis_package/deployers/utils.star` and `kurtosis_package/dockerfiles/contract_deployer.Dockerfile`.

Reply:
https://github.com/Layr-Labs/avs-devnet/pull/221/commits/f1d0c03e36ed2a141cbee940088dd0fa695901b2